src/core/infrastructure/guard.js

```diff
@@ -33,4 +33,15 @@ export function notNullOrEmpty(value, name){
 	if (value === null || isUndefined(value) || value === '') {
 		throw new AppError('guard.notNullOrEmpty', name);
 	}
+}
+
+/**
+ * If value is not a function exception will be thrown
+ * @param {any} value - Value to check
+ * @param {string} name - Argument name
+ */
+export function invokable(value, name){
```

Comment:
Yeah, I thought about that, but `export function function`. Probably I'll rewrite all exports